python/lsst/ip/isr/isrTask.py

```diff
@@ -224,12 +224,12 @@ class IsrTaskConfig(pexConfig.Config):
     fallbackFilterName = pexConfig.Field(dtype=str,
                                          doc="Fallback default filter name for calibrations", optional=True)
 
-## \addtogroup LSST_task_documentation
-## \{
-## \page IsrTask
-## \ref IsrTask_ "IsrTask"
-## \copybrief IsrTask
-## \}
+# \addtogroup LSST_task_documentation
+# \{
+# \page IsrTask
+# \ref IsrTask_ "IsrTask"
+# \copybrief IsrTask
+# \}
```

Comment:
This patch is not in the right commit. Surely it wasn't needed to make the tests pass.